CosNet.WebUI/CosNet.WebUI.csproj

```diff
@@ -30,6 +30,9 @@
   <ItemGroup>
     <ComponentScssFiles Include="**/*/*.scss" Exclude="node_modules/**;wwwroot/**;Styles/**" />
   </ItemGroup>
+  <ItemGroup>
+    <Watch Remove="Styles\_cosplay-item-show.scss" />
```

Comment:
deze watch remove voorkomt dat de scss gecompileerd zal worden, dus deze blok zou weg moeten denk k

Reply:
dat werd automatisch toegevoegd dus weet het zelf niet zeker. weet wel dat het wel werkt

Reply:
die wordt inderdaad automatisch toegevoegd als je een nieuw scss bestand maakt, maar dat moet weg, ik heb vorige week door dat een probleem mee gehad bij asp, daardoor dat ik het weet

Reply:
Ok

Reply:
Dat is inderdaad een fout van VS. VS Code heeft dat probleem niet. Alle SCSS files moeten worden geinclude in de watch om `dotnet watch run` correct te laten werken